action.yml

```diff
@@ -0,0 +1,106 @@
+name: Silverbackltd Build Action
+author: 'ApeWorX Ltd.'
+description: 'A GitHub Action to build docker images for a silverback repository and push to a container registry'
+branding:
+  icon: 'check-circle'
+  color: 'green'
+
+inputs:
+  push:
+    description: 'Push the image to ghcr'
+    required: false
+    default: 'false'
+  tag:
+    description: 'Tag of the silverback docker images'
+    required: false
+    default: 'latest'
+  registry:
+    description: 'The registry location you want to push to.'
+    required: false
+    default: 'ghcr.io'
+  username:
+    description: 'Add your username for your registry'
+    required: false
+    default: ${{ github.actor }}
+  password:
+    description: 'Add your password for your registry'
+    required: false
+    default: ''
```

Comment:
This shouldn't work with default, but then again it's probably not required unless you push right?

Reply:
correct

Reply:
I should add a check at the beginning of the job to check if push is true, then confirm there are values there. If there aren't give a descriptive error